lib/public/views/Tags/Details/TagDetailsModel.js

```diff
@@ -226,19 +236,45 @@ class TagDetailsTabbedPanelModel extends TabbedPanelModel {
         }
     }
 
+    /**
+     * Returns the pagination model
+     *
+     * @return {PaginationModel} the pagination model
+     */
+    get pagination() {
+        return this._pagination;
+    }
+
     /**
      * Fetches the logs related to the current tag
      *
+     * @param {boolean} clear if true, any previous data will be discarded, even in infinite mode
      * @returns {Promise<void>} resolves once the data has been retrieved
      * @private
      */
-    async _fetchLogsPanelData() {
+    async _fetchLogsPanelData(clear = false) {
+        /**
+         * @type {Run[]}
```

Comment:
Please update documentation as I believe here you are expecting Logs and not Runs